cli/clite/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "zip-clite"
+version = "0.0.1"
+authors = [
+    "Danny McClanahan <dmcC2@hypnicjerk.ai>",
+]
+license = "MIT"
+repository = "https://github.com/zip-rs/zip2.git"
+keywords = ["zip", "archive", "compression", "cli"]
+categories = ["command-line-utilities", "compression", "filesystem", "development-tools::build-utils"]
+rust-version = "1.74.0"
+description = """
+Binary for creation and manipulation of zip files.
+"""
+edition = "2021"
+
+# Prevent this from interfering with workspaces
+[workspace]
+members = ["."]
+
+[[bin]]
+name = "zip-clite"
+
+[dependencies]
+color-eyre = "0.6"
+
+[dependencies.zip-cli]
+path = ".."
+default-features = false
+features = ["deflate-flate2", "deflate-zlib"]
+
+[profile.release]
+strip = true
+lto = true
+opt-level = "s"
```

Comment:
It's actually larger!

```shell
# with opt-level = "s"
> ls target/release/zip-clite      
-rwxr-xr-x 2 cosmicexplorer wheel 780k Aug 14 15:58 target/release/zip-clite*
# with opt-level = "z"
> ls target/release/zip-clite
-rwxr-xr-x 2 cosmicexplorer wheel 801k Aug 17 13:47 target/release/zip-clite*
```

I also think there is a really good argument for *not* depending on `color-eyre` especially for the `clite` binary, since adding usages of `.wrap_err_with(...)` ended up adding a few hundred kb to the binary. I'm going to see if:
1. I can keep the same code but use the default `eyre` crate and get an improvement,
2. I can replace it with `anyhow` (and give up nice colored errors) and get an improvement,
3. I can do like I did with `clap` and produce my own tiny `WrapErr` trait and associated `Error` object that avoids calling into other libraries.

I think as with `clap`, since we're not making a really hulking server application with tons of error cases, (3) is not actually a bad idea. When I tried something like (3) previously, I didn't try making something like a `WrapErr` trait, but methods I had to call manually which was rather annoying and difficult to extend. I think doing it that way is probably a good idea, honestly.